runners/apex/src/main/java/org/apache/beam/runners/apex/translation/operators/ApexGroupByKeyOperator.java

```diff
@@ -197,7 +198,7 @@ public void outputWindowedValue(
           }
         },
         NullSideInputReader.empty(),
-        null,
```

Comment:
Can you say more about this change? Is this a bug in Apex? Does not seem Aggregator-related.